src/h3lib/lib/h3Index.c

```diff
@@ -784,7 +784,7 @@ H3Error H3_EXPORT(latLngToCell)(const LatLng *g, int res, H3Index *out) {
     if (*out) {
         return E_SUCCESS;
     } else {
-        return E_FAILED;
+        return E_FAILED;  // todo: under what circumstances would this failure happen?
```

Comment:
It's not clear to me - looks like it will fail if the `faceIjk` object has out-of-bounds coordinates, but I can't easily tell whether that's a real case with bad coordinates or a fail-safe.